js/__tests__/helper.test.js

```diff
@@ -1,38 +1,38 @@
 /* eslint-env jest */
 
-var helper = require('../helper');
+const helper = require('../helper');
 
-var SOURCEMAPS_OPTIONS = require('../releases/options/uploadSourcemaps');
+const SOURCEMAPS_OPTIONS = require('../releases/options/uploadSourcemaps');
 
-describe('SentryCli helper', function() {
-  test('call sentry-cli --version', function() {
+describe('SentryCli helper', () => {
+  test('call sentry-cli --version', () => {
     expect.assertions(1);
-    return helper.execute(['--version']).then(function() {
-      expect(true).toBe(true);
-    });
+    return helper
+      .execute(['--version'])
+      .then(version => expect(version.trim()).toBe('sentry-cli DEV'));
```

Comment:
Added a check for the output. Note the `trim()` - we could consider trimming in `helper.execute()`